coderd/notifications/metrics_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/coder/quartz"
+
 	"github.com/coder/serpent"
```

Comment:
Another instance of import grouping, this is the job of the tooling tbh (and it's failing), so up to you if you want to fix it 😄. (There are actually a few more of these too but I'll leave those uncommented.)

Reply:
I'll leave this for now; we need the tooling to be updated as you suggest.
Thanks for pointing this out though